prow/plugins/lgtm/lgtm.go

```diff
@@ -204,3 +207,26 @@ func handle(gc githubClient, log *logrus.Entry, e *event) error {
 	}
 	return nil
 }
+
+func handlePullRequest(gc githubClient, pe github.PullRequestEvent) error {
+	if pe.PullRequest.Merged {
+		return nil
+	}
+
+	switch pe.Action {
+	case "synchronize":
+		// Don't bother checking if it has the label...it's a race, and we'll have
+		// to handle failure due to not being labeled anyway.
+		if err := gc.RemoveLabel(
+			pe.PullRequest.Base.Repo.Owner.Login,
+			pe.PullRequest.Base.Repo.Name,
+			pe.PullRequest.Number,
+			lgtmLabel,
+		); err != nil {
+			// TODO(nlandolfi): inspect this error?
```

Comment:
What does the API return if you ask it to remove a label from a PR that doesn't have that label? Try it out on a test PR.